From 1365d429f46f9684850ed3188f5d079ef5e92e14 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 16 Jul 2020 07:42:57 -0500
Subject: [PATCH 17/23] board: imx8m/n beacon: Fix scripts to generate proper
 flash.bin

If compiling from uboot-imx, the user needs to go to ../imx-mkimage
to find the flash.bin file.

This patch adds a copy instruction to move the flash.bin file back
to uboot when generated.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 txfrToMkImg-M.sh | 1 +
 txfrToMkImg-N.sh | 1 +
 2 files changed, 2 insertions(+)

diff --git a/txfrToMkImg-M.sh b/txfrToMkImg-M.sh
index 2e8c004331..e99050b67b 100755
--- a/txfrToMkImg-M.sh
+++ b/txfrToMkImg-M.sh
@@ -7,5 +7,6 @@ cp ./spl/u-boot-spl.bin ../imx-mkimage/iMX8M
 cp ./spl/u-boot-spl-nodtb.bin ../imx-mkimage/iMX8M
 cd ../imx-mkimage
 make SOC=iMX8MM flash_hdmi_spl_uboot
+cp iMX8M/flash.bin ../uboot-imx/.
 cd ../uboot-imx
 
diff --git a/txfrToMkImg-N.sh b/txfrToMkImg-N.sh
index f18af6686f..345dab4def 100755
--- a/txfrToMkImg-N.sh
+++ b/txfrToMkImg-N.sh
@@ -8,5 +8,6 @@ cp ./spl/u-boot-spl.bin ../imx-mkimage/iMX8M
 cp ./spl/u-boot-spl-nodtb.bin ../imx-mkimage/iMX8M
 cd ../imx-mkimage
 make SOC=iMX8MN flash_hdmi_spl_uboot
+cp iMX8M/flash.bin ../uboot-imx/.
 cd ../uboot-imx
 
-- 
2.17.1

